tools/report-converter/README.md

```diff
@@ -532,6 +535,31 @@ report-converter -t sparse -o ./codechecker_sparse_reports ./sparse.out
 CodeChecker store ./codechecker_sparse_reports -n sparse
 ```
 
+## [cpplint](https://github.com/cpplint/cpplint)
+[cpplint](https://github.com/cpplint/cpplint) is a lint-like tool which checks
+C++ code against [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html).
+
+The recommended way of running cpplint is to redirect the output to a file and
+give this file to the report converter tool.
+
+The following example shows you how to run cpplint and store the results
+found by cpplint to the CodeChecker database.
+
+```sh
+# Change Directory to your project
+cd path/to/linux/kernel/repository
```

Comment:
cpplint is not a kernel specific analyzer, so we need to change this line to something like this:
```suggestion
cd path/to/your/project
```